src/huggingface_hub/repository.py

```diff
@@ -554,14 +654,13 @@ def git_pull(self, rebase: Optional[bool] = False):
         if rebase:
             args.append("--rebase")
         try:
-            subprocess.run(
-                args,
-                stderr=subprocess.PIPE,
-                stdout=subprocess.PIPE,
-                check=True,
-                encoding="utf-8",
-                cwd=self.local_dir,
-            )
+            with lfs_log_progress():
+                subprocess.run(
+                    args,
+                    check=True,
+                    encoding="utf-8",
+                    cwd=self.local_dir,
+                )
```

Comment:
This really can't fit in one line (at least all the args on a new line?). Not used to 80 chars max.

Reply:
I'm contemplating switching the default to be the same as `transformers`' on a daily basis :)

Reply:
You definitely should!